mev_inspect/coinbase.py

```diff
@@ -0,0 +1,25 @@
+import aiohttp
+
+from mev_inspect.classifiers.specs.weth import WETH_ADDRESS
+from mev_inspect.schemas.transfers import ETH_TOKEN_ADDRESS
+from mev_inspect.schemas.coinbase import CoinbasePrices, CoinbasePricesResponse
+
+
+COINBASE_API_BASE = "https://www.coinbase.com/api/v2"
+COINBASE_TOKEN_NAMES = {
+    WETH_ADDRESS: "weth",
```

Comment:
Why are these called addresses? A bit confusing because they aren't eth addresses

Reply:
The dict itself is address => "coinbase name". The `WETH_ADDRESS` is an address

I should probably rename to `COINBASE_TOKEN_NAME_BY_ADDRESS`

Thoughts?

Reply:
That makes sense to me

Reply:
Updated - marking resolves